utils/awsFunctions.ts

```diff
@@ -0,0 +1,173 @@
+import {
+    IdentitystoreClient,
+    ListUsersCommand,
+    CreateUserCommand,
+    CreateGroupMembershipCommand,
+    ListUsersCommandInput,
+    CreateUserCommandInput,
+    CreateGroupMembershipCommandInput,
+  } from "@aws-sdk/client-identitystore";
+  import config from "config";
+  
+  // Define the configuration variables with proper types
+  const accessKeyId: string = config.get<string>("aws.access_key");
+  const secretAccessKey: string = config.get<string>("aws.secret_key");
+  const region: string = config.get<string>("aws.region");
+  const identityStoreId: string = config.get<string>("aws.identity_store_id");
+  
+  let client: IdentitystoreClient;
```

Comment:
Instead of calling `configureAWSCredentials()` method in each function, can we initialise this variable with the client?
Like this
```suggestion
  const client: IdentitystoreClient = configureAWSCredentials();
```

Reply:
Here we are lazily initialising the client and returning the same client in case once we have created it @samarpan1738.